pkg/deploy/util/util.go

```diff
@@ -18,10 +18,16 @@ import (
 	kcoreclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
 	kdeplutil "k8s.io/kubernetes/pkg/controller/deployment/util"
 
+	osapiv1 "github.com/openshift/origin/pkg/api/v1"
 	deployapi "github.com/openshift/origin/pkg/deploy/api"
 	"github.com/openshift/origin/pkg/util/namer"
 )
 
+var (
+	// controllerKind contains the schema.GroupVersionKind for this controller type.
+	ControllerKind = osapiv1.SchemeGroupVersion.WithKind("DeploymentConfig")
```

Comment:
@kargakis will this break with legacy kind? (no api group?)

Also do we need this exported? If yes, fix the godoc. Also I would rename this to `ownerRefControllerKind` or something to be less generic. 

maybe we don't need this at all if we use this only in one place...